.github/CODEOWNERS

```diff
@@ -1,5 +1,3 @@
 # AVM core team owns key files
 .github/policies/ @Azure/avm-core-team-technical
 .github/CODEOWNERS @Azure/avm-core-team-technical
-* @Azure/terraform-azurerm-avm-ptn-aks-production-module-owners-tf
```

Comment:
@matt-FFFFFF do we need to remove this, i'm assuming the grept check didn't mean to change this files